examples/async.php

```diff
@@ -6,57 +6,56 @@
 # 10 minutes. This is useful when creating many documents in parallel, or very
 # large documents with lots of assets.
 #
-# DocRaptor supports many options for output customization, the full list is
-# https://docraptor.com/documentation/api#api_general
+# DocRaptor supports many CSS and API options for output customization. Visit
+# https://docraptor.com/documentation/ for full details.
 #
 # You can run this example with: php async.php
-require __DIR__."/../vendor/autoload.php";
+
+require_once __DIR__ . "/vendor/autoload.php";
```

Comment:
I know you said the tests pass, but this path change seems like it would break running the tests in this repo?

Reply:
This doesn't affect the [tests](https://github.com/DocRaptor/docraptor-php/tree/master/test) at all. But yes, it's not necessarily the correct path for all scenarios. It assumes the example script is at the root of your PHP directory; as if you just installed the agent and are now immediately trying to use the script.

Reply:
To be more specific, I meant the tutorial-generator tests. I didn't run (or change in any way) the tests or code in this repo.

Reply:
Ah yeah, that makes sense.